app/models/repair.rb

```diff
@@ -4,9 +4,14 @@ class Repair < ActiveRecord::Base
                   :status, :user_id, :cost
   belongs_to :item
   belongs_to :person
-  validates_numericality_of :cost
+  delegate :item_category, :to => :item
+  delegate :item_type, :to => :item
```

Comment:
i ... don't really understand why these exist on the repair even if they're delegated. I don't see why the repair cares what "type" of item. 

For example. if the "repair" is fixing a wheel it doesn't matter so much what _kind_ of car it is that gets repaired. it matters _which_ car it is.